Formula/a/azure-cli.rb

```diff
@@ -30,7 +30,7 @@ class AzureCli < Formula
   depends_on "libsodium"
   depends_on "libyaml"
   depends_on "openssl@3"
-  depends_on "python@3.12"
+  depends_on "python@3.11"
```

Comment:
maybe good to also add a PR or issue in here for future ref

Reply:
There isn't one currently, but I've been talking to upstream, so will follow up.